.rubocop.yml

```diff
@@ -13,6 +13,8 @@ RSpec/NestedGroups:
   Enabled: false
 RSpec/ContextWording:
   Enabled: false
+Style/OptionalBooleanParameter:
```

Comment:
ill do a little more research on the changes made here and then ill start chipping away at it2